maestro-client/src/main/java/maestro/drivers/AndroidDriver.kt

```diff
@@ -158,6 +158,10 @@ class AndroidDriver(
         ) ?: throw IllegalStateException("Response can't be null")
     }
 
+    override fun longPress(point: Point) {
+        dadb.shell("input swipe ${point.x} ${point.y} ${point.x} ${point.y} 3000")
```

Comment:
FYI 4.4 and above supports this:
https://stackoverflow.com/a/27630157

Reply:
Saw it, but it is for key events, not taps

Reply:
ah makes sense